src/rpc/client/ts/runTests.ts

```diff
@@ -0,0 +1,69 @@
+const { execSync, spawn } = require('child_process')
+const { exit } = require('process')
+const glob = require('glob')
+const fs = require('fs')
+const os = require('os')
+const unzip = require('unzip-stream')
+
+// Extract server
+async function extractServer(filePath) {
+  await new Promise((res, rej) => {
+    let stream = fs
+      .createReadStream(`${filePath}.zip`)
+      .pipe(unzip.Extract({ path: '.' }))
+    stream.on('close', () => {
+      try {
+        res(`${filePath}.zip`)
+      } catch (err) {
+        rej(err)
+      }
+    })
+  })
+}
+
+// Run Scala gRPC server
+async function startServer(filePath) {
+  if (!os.platform().toLowerCase().startsWith('win')) {
+    execSync(`chmod +x ${filePath}/bin/omega-edit-grpc-server`)
```

Comment:
Probably want to quote that file path (e.g., chmod +x "${filePath}/bin/omega-edit-grpc-server").

Reply:
I don' think it matters that it is quoted or not. But I will fix the issue with execSync